packages/xo-web/src/xo-app/user/index.js

```diff
@@ -282,6 +289,85 @@ const SshKeys = addSubscriptions({
   )
 })
 
+// ===================================================================
+const dateFormat = d3.timeFormat('%Y-%m-%d %H:%M')
+
+const COLUMNS_AUTH_TOKENS = [
+  {
+    itemRenderer: ({ id }) => (
+      <Copiable tagName='pre' data={id}>
+        {id.slice(4, 9)}
```

Comment:
Maybe we should add a ellipsis suffix?

```suggestion
        {id.slice(4, 9)}…
```

What's your opinion on this?

Reply:
indeed, it avoids the user to think that his token is only the 5 characters displayed.